modules/events_archive/EED_Events_Archive.module.php

```diff
@@ -841,45 +796,44 @@ public static function update_template_settings($CFG, $REQ)
             ! isset($REQ['EED_Events_Archive_reset_event_list_settings'])
             || absint($REQ['EED_Events_Archive_reset_event_list_settings']) !== 1
         ) {
-            $CFG->EED_Events_Archive->display_status_banner = isset($REQ['EED_Events_Archive_display_status_banner'])
+            $CFG->EED_Events_Archive->display_status_banner   = isset($REQ['EED_Events_Archive_display_status_banner'])
                 ? absint($REQ['EED_Events_Archive_display_status_banner']) : 0;
-            $CFG->EED_Events_Archive->display_description = isset($REQ['EED_Events_Archive_display_description'])
+            $CFG->EED_Events_Archive->display_description     = isset($REQ['EED_Events_Archive_display_description'])
                 ? absint($REQ['EED_Events_Archive_display_description']) : 1;
-            $CFG->EED_Events_Archive->display_ticket_selector = isset($REQ['EED_Events_Archive_display_ticket_selector'])
-                ? absint($REQ['EED_Events_Archive_display_ticket_selector']) : 0;
-            $CFG->EED_Events_Archive->display_datetimes = isset($REQ['EED_Events_Archive_display_datetimes']) ? absint(
-                $REQ['EED_Events_Archive_display_datetimes']
-            ) : 1;
-            $CFG->EED_Events_Archive->display_venue = isset($REQ['EED_Events_Archive_display_venue']) ? absint(
-                $REQ['EED_Events_Archive_display_venue']
-            ) : 0;
-            $CFG->EED_Events_Archive->display_expired_events = isset($REQ['EED_Events_Archive_display_expired_events'])
+            $CFG->EED_Events_Archive->display_ticket_selector =
+                isset($REQ['EED_Events_Archive_display_ticket_selector'])
+                    ? absint($REQ['EED_Events_Archive_display_ticket_selector']) : 0;
+            $CFG->EED_Events_Archive->display_datetimes       =
+                isset($REQ['EED_Events_Archive_display_datetimes']) ? absint(
+                    $REQ['EED_Events_Archive_display_datetimes']
+                ) : 1;
+            $CFG->EED_Events_Archive->display_venue           =
+                isset($REQ['EED_Events_Archive_display_venue']) ? absint(
+                    $REQ['EED_Events_Archive_display_venue']
+                ) : 0;
+            $CFG->EED_Events_Archive->display_expired_events  = isset($REQ['EED_Events_Archive_display_expired_events'])
                 ? absint($REQ['EED_Events_Archive_display_expired_events']) : 0;
         }
         return $CFG;
     }
 
```

Comment:
I see different coding styles for this function for the same logic, I think it is better to have one for all.

Reply:
ahhh ya... autoformatting doesn't always work great and i missed that. 

thnx will fix